packages/react/src/components/Masthead/__tests__/MastheadTopNav.test.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { settings as ddsSettings } from '@carbon/ibmdotcom-utilities';
+import MastheadTopNav from '../MastheadTopNav';
+import { mount } from 'enzyme';
+import React from 'react';
+import { TranslationAPI } from '@carbon/ibmdotcom-services';
+
+const { stablePrefix } = ddsSettings;
+
+describe('MastheadTopNav', () => {
+  it('renders one MastheadTopNav', () => {
+    const wrapper = mount(<MastheadTopNav navigation={[]} />);
+
+    expect(wrapper).toHaveLength(1);
+  });
+
+  it('renders all the itens based in the `navigation` prop', async () => {
+    const data = await TranslationAPI.getTranslation();
+    const { links } = data.mastheadNav;
+    const wrapper = mount(<MastheadTopNav navigation={links} />);
+    const menuItems = links.map((_itens, index) => {
+      return wrapper.find(
+        `a[data-autoid="${stablePrefix}--masthead__l0-nav--nav-${index}"]`
+      );
+    });
+
+    expect(menuItems).toHaveLength(links.length);
+  }, 15000);
```

Comment:
@guilhermelMoraes What is this for?

Reply:
@kennylam this third parameter is a timeout for the `it` function. Since we're fetching the api and not only mocking it, the timeout is an assurance that the test will not fail due to the api takes too long to respond

Reply:
hmm you should be able to mock like so:
```
jest.mock('@carbon/ibmdotcom-services', () => ({
  TranslationAPI: {
    getTranslation: jest.fn(() => Promise.resolve(MOCK_DATA)),
  },
}))
```
with `MOCK_DATA` being the example data you see (I pulled this out of Footer.test.js)

This way we don't have to make the actual call